README.md

```diff
@@ -3,30 +3,42 @@ About tzlocal
 
 Home: https://github.com/regebro/tzlocal
 
-Package license: CC0 1.0 Universal
+Package license: LICENSE.txt
```

Comment:
Appears to be caused by a [typo]( https://github.com/conda-forge/tzlocal-feedstock/commit/36188abbb8b8a293b2317701e3c37c5d073cec4f#diff-e178b687b10a71a3348107ae3154e44cR39 ).